src/Core/src/Platform/Windows/ButtonExtensions.cs

```diff
@@ -159,6 +159,7 @@ public static void UpdateImageSource(this Button platformButton, WImageSource? n
 		{
 			if (platformButton.GetContent<WImage>() is WImage nativeImage)
 			{
+				nativeImage.Stretch = UI.Xaml.Media.Stretch.Uniform;
```

Comment:
This may be different now as there was a major change in https://github.com/dotnet/maui/pull/17159 to make this all stretch. I see further on there is already this line, so this line can be removed. But, not sure why/if the removed code will also affect this PR.

Reply:
This line is duplicated. See lines 166 and 167:

https://github.com/dotnet/maui/pull/19834/files#diff-e34feff8430d43e27f52abff099ba347b41628f6bde6a68e12f8b618722c59c9R166-R167

Reply:
Image stretch and `HorizontalAlignment` and `VerticalAlignment` weirdly enough act differently here. `Stretch` looks like the "right" one to use vs alignment.